### ISSUE TITLE
K8s operator

### ISSUE THREAD

**Issue by frotsch** [Yelp/nrtsearch#397]
Any plans to open source the K8s operator mentioned in https://engineeringblog.yelp.com/2021/09/nrtsearch-yelps-fast-scalable-and-cost-effective-search-engine.html ?

**Comment by umeshdangat** [comment]
Yes we do intend to open source a version of the k8s operator that we are currently using internally at Yelp. However we do not have a definitive timeline when we would get around to this. 

In its current form, there are some quirks that we are looking at simplifying before we can open source this. For example our usage of stateful sets and specific k8s [provisioner](https://github.com/kubernetes/examples/blob/master/staging/persistent-volume-provisioning/README.md#aws). Ideally we do not really need to rely on stateful sets.

**Comment by frotsch** [comment]
That sounds great :-)
